auth/auth_test.go

```diff
@@ -49,7 +57,7 @@ func TestToken(t *testing.T) {
 	}
 
 	// Obtain info
-	req, err := http.NewRequest("GET", "https://auth.arduino.cc/v1/users/byID/me", nil)
+	req, err := http.NewRequest("GET", "https://ddauth.arduino.cc/v1/users/byID/me", nil)
```

Comment:
I think this is a typo

Reply:
good catch!

Reply:
(for context, tests are green because this specific test is skipped on Travis, found out when it was too late)